source/Body.cpp

```diff
@@ -352,10 +352,10 @@ void Body::Turn(double amount)
 		return;
 
 	auto RotatePointAroundOrigin = [](Point &toRotate, double radians) -> Point {
-		float si = sin(radians);
-		float co = cos(radians);
-		float newX = toRotate.X() * co - toRotate.Y() * si;
-		float newY = toRotate.X() * si + toRotate.Y() * co;
+		double si = sin(radians);
+		double co = cos(radians);
+		double newX = toRotate.X() * co - toRotate.Y() * si;
+		double newY = toRotate.X() * si + toRotate.Y() * co;
```

Comment:
There is a cost to converting between double and float, but it's not significant. The real cost is in the `sin` and `cos` functions.

If we change `radians` to a float, this could could be micro-benchmarked to be significantly faster. Even then, it wouldn't have a measurable impact on the game's performance.

Reply:
no, but I was debugging much more serious performance problems and spotted this.
sinf() and cosf() would be better, but also we could have Pointf().

Reply:
oh yeah, I forget all this C++ stuff sometimes...
but I see no benefit to using an overloaded function vs. a non-overloaded one.